documentation/advanced-creation.md

```diff
@@ -0,0 +1,238 @@
+# Advanced creation
+
+> Make calling REST APIs easier by creating niche-specific `got` instances.
+
+#### got.create(settings)
+
+Example: [gh-got](https://github.com/sindresorhus/gh-got/blob/master/index.js)
+
+Configures a new `got` instance with the provided settings. You can access the resolved options with the `.defaults` property on the instance.
+
+**Note:** In contrast to [`got.extend()`](../readme.md#gotextendinstances), this method has no defaults.
+
+##### [options](readme.md#options)
+
+To inherit from parent, set it to `got.defaults.options` or use [`got.mergeOptions(defaults.options, options)`](../readme.md#gotmergeoptionsparentoptions-newoptions).<br>
+**Note:** Avoid using [object spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals) as it doesn't work recursively.
+
+**Note #2:** [`got.mergeOptions()`](../readme.md#gotmergeoptionsparentoptions-newoptions) does not merge hooks. Use [`got.extend()`](../readme.md#gotextendinstances) instead.
+
+##### mutableDefaults
+
+Type: `boolean`<br>
+Default: `false`
+
+States if the defaults are mutable. It's very useful when you need to [update headers over time](readme.md#hooksafterresponse), e.g. update the access token when it expires.
```

Comment:
```suggestion
States if the defaults are mutable. It can be useful when you need to [update headers over time](readme.md#hooksafterresponse), for example, update an access token when it expires.
```